File: Python/MathematicalPrograms/3DShapes/VolumeOfCube.py
ques=input('Do you wish to find the Volume of Cube by 2D Method or via Side Method? Please enter either 2D or Side')
if ques=='2D':
    print ('OK.')
    ba=int(input('Enter the Base Area of the Cube'))
    Height=int(input('Enter a Height measure'))
    v=ba*Height
    print ('Volume of the Cube is', v)
elif ques=='Side':
    print ('OK.')
    side=int(input('Enter the measure of the Side'))
    v=side**3
    print ('Volume of the Cube is', v)
else:
    exit()

File: C++/MathematicalPrograms/3DShapes/VolumeOfCube.cpp
#include <iostream>
#include <string>
#include <math.h>

using namespace std;

int main() {
	string answer;
	int side;
	int ba, height;
	int volume;
	cout << "Do you wish to find the Volume of Cube by 2D Method or via Side Method? Please enter either 2D or Side";
	getline(cin, answer);
	if (answer == "2D") {
		cout << "OK.";
		cout << "\nEnter the Base Area of the Cube";
		cin >> ba;
		cout << "Enter a Height measure";
		cin >> height;
		volume = ba*height;
		cout << "Volume of the Cube is " << volume;
	}
	else if (answer == "Side") {
		cout << "OK.";
		cout << "\nEnter the measure of the Side";
		cin >> side;
		volume = pow(side, 3);
		cout << "Volume of the Cube is " << volume;
	}
	else {
		exit(0);
	}
}